### PR TITLE
Fix wording in the installer

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Obviously this message should be about the `mod_auth_basic`
| Type?             | bug fix
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | n/a
| How to test?      | n/a
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.
| Sponsor company | impSolutions

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26591)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by tegbessou — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

File: src/Core/Webservice/ServerRequirementsChecker.php [hunk 1]
```diff
@@ -149,7 +149,7 @@ private function getWarningMessages()
     {
         return [
             self::ISSUE_APACHE_MOD_AUTH_BASIC_NOT_AVAILABLE => $this->translator->trans(
-                'Please activate the \'mod_rewrite\' Apache module to allow the PrestaShop webservice.',
+                'Please activate the \'mod_auth_basic\' Apache module to allow the use of the PrestaShop webservice.',
```
Comment (Julievrz) [inline src/Core/Webservice/ServerRequirementsChecker.php:152]: Hi @kpodemski 

If this wording appears in the installer, then the translation domain should be "Install". :-)

Comment (kpodemski) [reply src/Core/Webservice/ServerRequirementsChecker.php:152]: Hello @Julievrz 

Could you tell me the entire domain that you would suggest? Right now, all wording in the installer is within the same domain.

https://github.com/PrestaShop/PrestaShop/blob/451b19538e51f434f4d5bcd266f122f64fbe723f/src/Core/Webservice/ServerRequirementsChecker.php#L150

Comment (Julievrz) [reply src/Core/Webservice/ServerRequirementsChecker.php:152]: Actually, the full domain is "Install". 😊 You can have a look here: https://zeroheight.com/80a6c5a61/v/latest/p/738e72-manage-translation-domains

Comment (kpodemski) [reply src/Core/Webservice/ServerRequirementsChecker.php:152]: I've just changed the domain to `Install`, thanks 👍

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @kpodemski

**Pull request merged**